### PR TITLE
feat: make js-feature  non default

### LINKED ISSUE #10441: Make js-feature non default
### Describe the feature

js-tracer is currently always enabled

https://github.com/paradigmxyz/reth/blob/f4255216c78e78ae94ca8927f9f8f2afccb0d69c/crates/rpc/rpc/Cargo.toml#L30-L30

we can avoid this by
1. make this non-default: https://github.com/paradigmxyz/reth/blob/f4255216c78e78ae94ca8927f9f8f2afccb0d69c/crates/rpc/rpc-eth-types/Cargo.toml#L62-L62
2. introduce js-tracer feature on rpc that enables: https://github.com/paradigmxyz/reth/blob/f4255216c78e78ae94ca8927f9f8f2afccb0d69c/crates/rpc/rpc/Cargo.toml#L30-L30 and feature gate support for js tracer

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by nkysg**
closes https://github.com/paradigmxyz/reth/issues/10441

### CODE REVIEW

File: crates/rpc/rpc/src/debug.rs [hunk 1]
```diff
@@ -33,9 +33,10 @@ use revm::{
     primitives::{db::DatabaseCommit, BlockEnv, CfgEnvWithHandlerCfg, Env, EnvWithHandlerCfg},
     StateBuilder,
 };
+#[cfg(feature = "js-tracer")]
+use revm_inspectors::tracing::js::JsInspector;
```
Comment (onbjerg) [inline crates/rpc/rpc/src/debug.rs:37]: let's change this import to not be feature gated and instead use the fully qualified path down below, ref #10440

**Review by onbjerg — CHANGES_REQUESTED** [review]

File: crates/rpc/rpc/src/debug.rs (hunk 1 above)
Comment (onbjerg) [reply crates/rpc/rpc/src/debug.rs:37]: the change has not been pushed:)

**Review by onbjerg — APPROVED** [review]

**Comment by onbjerg** [comment]
thank you!

**Review by mattsse — APPROVED** [review]
nice, I've enabled js-tracer in rpc-builder now otherwise this would now no longer be enabled when we launch the node

**Pull request merged**